main.py

```diff
@@ -219,6 +220,24 @@ def ui_access_wrapper(self, *args, **kwargs):
     get_redirect_route(
         r'%s/remove_in_progress_collection' % feconf.LEARNER_DASHBOARD_URL,
         reader.RemoveCollectionFromIncompleteListHandler),
+
+    get_redirect_route(
+        r'%s/add_exploration_to_learner_playlist' %
```

Comment:
Although there exists a POST method, there exists no such method for delete functionality. I tried creating a function in test_utils.py, replicating the already existing functions like this - 
```
    def delete_json(self, url, payload, csrf_token=None, expect_errors=False,
                    expected_status_int=200):
        data = {'payload': json.dumps(payload)}
        if csrf_token:
            data['csrf_token'] = csrf_token

        json_response = self.testapp.delete(
            str(url), data, expect_errors=expect_errors)

        self.assertEqual(json_response.status_int, expected_status_int)
        return self._parse_json_response(
            json_response, expect_errors=expect_errors)
```
however, this is not working for some reason. The error it gives is internal server error 500. Could you please help me with this? I have very little experience with this.